### PR TITLE
added localstack s3 buckets for local environment

### PR DESCRIPTION
**Pull request by Marmik2003**
1. added localstack in docker-compose.local.yaml
2. added bucket-setup.sh to create buckets upon running container

### CODE REVIEW

**Comment by Marmik2003** [comment]
fixes #751

**Review by sainAk — APPROVED** [review]

File: docker-compose.local.yaml [hunk 1]
```diff
@@ -27,5 +27,39 @@ services:
...
+    image: localstack/localstack:latest
+    environment: 
+      - AWS_DEFAULT_REGION=ap-south-1
+      - EDGE_PORT=4566
+      - SERVICES=s3
+    ports: 
+      - '4566-4583:4566-4583'
+    volumes: 
+      - "${TEMPDIR:-/tmp/localstack}:/tmp/localstack"
+      - "/var/run/docker.sock:/var/run/docker.sock"
```
Comment (vigneshhari) [inline docker-compose.local.yaml:61]: We don't need to mount docker.

**Review by vigneshhari — CHANGES_REQUESTED** [review]

File: docker-compose.local.yaml [hunk 2]
```diff
@@ -27,5 +27,39 @@ services:
...
+      - "6379:6379"
+
+  celery:
+    container_name: care_celery
+    build: 
+      context: .
+      dockerfile: docker/DevDockerfile
+    env_file:
+      - ./docker/.local.env
+    entrypoint: ["bash", "devCelery.sh"]
```
Comment (vigneshhari) [inline docker-compose.local.yaml:44]: Since this is the local version, we should mount local volumes as well.

**Review by vigneshhari — CHANGES_REQUESTED** [review]

File: docker-compose.local.yaml (hunk 1 above)
Comment (Marmik2003) [reply docker-compose.local.yaml]: done

File: docker-compose.local.yaml (hunk 2 above)
Comment (Marmik2003) [reply docker-compose.local.yaml]: done

File: docker-compose.local.yaml [hunk 3]
```diff
@@ -27,5 +27,39 @@ services:
...
+    image: redis:alpine
+    restart: always
+    ports:
+      - "6379:6379"
+
+  celery:
+    container_name: care_celery
+    build:
+      context: .
+      dockerfile: docker/DevDockerfile
```
Comment (vigneshhari) [inline docker-compose.local.yaml:41]: This does not contain a beat instance, create another container for beat or add the beat functionality to this container.

**Review by vigneshhari — CHANGES_REQUESTED** [review]

**Review by vigneshhari — APPROVED** [review]
LGTM. Have not verified.

**Pull request merged**